### PR TITLE
Updates to Ed API

### PR DESCRIPTION
**Pull request by wortzmanb**
Updates that are used in the quiz generation script.

### CODE REVIEW

File: teachingtoolshed/api/edstem.py [hunk 1]
```diff
@@ -116,6 +162,112 @@ def get_lesson(self, lesson_id: int) -> Dict[str, Any]:
...
+
+        clone_path = urljoin(EdStemAPI.API_URL, f"lessons/slides/{slide_id}/clone")
+        payload = {
+            "lesson_id": lesson_id,
+            "is_hidden": is_hidden
+        }
+        slide = json.loads(self._ed_post_request(clone_path, json=payload))["slide"]
+        return slide
+
+    # def edit_slide(
```
Comment (hschafer) [inline teachingtoolshed/api/edstem.py:247]: What is the point of including this?

Comment (wortzmanb) [reply teachingtoolshed/api/edstem.py:247]: I eventually want to be able to edit slides in the API (mostly to change titles), but it doesn't work yet. Fine to leave it out until I get it working.

Comment (wortzmanb) [reply teachingtoolshed/api/edstem.py:247]: Let me know if you want me to change the PR. I think you can do it, though?

Comment (hschafer) [reply teachingtoolshed/api/edstem.py:247]: Eh it's fine to leave it then.

**Pull request merged**